doc/guide/guide-states.rst

```diff
@@ -843,10 +844,10 @@ This isomorphism is implemented in QuTiP by the
     np.testing.assert_almost_equal((rho - rho2).norm(), 0)
 
 The :attr:`~qutip.Qobj.type` attribute indicates whether a quantum object is
-a vector corresponding to an operator (``operator-ket``), or its Hermitian
+a vector corresponding to an operator (``operator-ket``), or to its Hermitian
 conjugate (``operator-bra``).
```

Comment:
This whole sentence feels wrong, `type` can indicate a lot more than just those 2 cases, but it is worded as this is all it can do...